source/clear-linux/get-started/virtual-machine-install/vmware-player-2.rst

```diff
@@ -0,0 +1,469 @@
+
+For additional help, see the `VMware Workstation Player guide`_.
+
+Download the latest |CL| image
+==============================
+
+Go to the |CL| `image`_ repository and download the desired type:
+
+* ISO installer image: `clear-<version>-installer.iso.xz` (for Method 1)
+* VMware image: `clear-<version>-basic.vmdk.xz` (for Method 2)
+
+For older versions, see the `releases`_ page.
+
+Although not required, it is recommended to download the corresponding 
+checksum file (designated with `-SHA512SUMS` at the end of the filename) 
+for the image in order to verify its integrity.
+
+Verify the integrity of the download (recommended)
+==================================================
+
+* For Linux distros:
+
+  #.  Start a terminal emulator.
+  #.  Go to the directory with the downloaded files.
+  #.  To verify the integrity of the image, enter the following (an installer ISO
+      image is used as an example):
+
+      .. code-block:: console
+
+        $ sha512sum ./clear-<version>-installer.iso.xz | diff ./clear-<version>-installer.iso.xz-SHA512SUMS -
+
+      If the checksum of the downloaded image is different than the original
+      checksum, the differences will displayed. An empty output indicates a match.
+
+* For Windows:
+
+  #.  Start Command-Prompt.
+  #.  Go to the directory with the downloaded files.
+  #.  To verify the integrity of the image, enter the following commands:
+
+      .. code-block:: console
+
+        C:\> CertUtil -hashfile ./clear-<version>-installer.iso.xz | findstr -v file
+
+      Compare the output with the original checksum to make sure they match.
+
+Uncompress the image
+====================
+
+* For Linux distros (an installer ISO image is used as an example):
+
+  .. code-block:: console
+
+    $ unxz clear-<version>-installer.iso.xz
+
+* For Windows:
+
+  Use `7zip`_ to uncompress it.
+
+Method 1: Fresh installation of |CL| into a new VM 
+==================================================
+
+The general process for performing a fresh installation of |CL| into a new VM 
+is as follows (with expanded details below):
+
+* Create a new VM and configure its settings
+* Attach the installer ISO to it
+* Install |CL|
+* Detach the installer ISO
+* Power off the VM
+* Enable EFI boot support
+* Power on the VM
+
+Create and configure a new VM 
+*****************************
+
+#.  Start the `VMware Workstation Player` app.
+#.  On the home screen:
+
+    * Click `Create a New Virtual Machine`.
+
+      |vmware-player-01|
+
+      Figure 1: VMware Workstation 14 Player - Create a new virtual machine
+   
+#.  On the `Welcome to the New Virtual Machine Wizard` step:
+
+    * Select `Installer disc image file (iso)`.
+    * Click `Browse` and select the uncompressed |CL| installer ISO. 
+    * Click `Next`.
+
+      |vmware-player-02|
+
+      Figure 2: VMware Workstation 14 Player - Select |CL| installer ISO   
+
+#.  On the `Select a Guest Operating System` step:
```

Comment:
Since these steps have descriptive titles, we can go ahead and just have this instruction number be named whatever the step is:
#. Select a guest operating system:
...
#. Name the virtual machine:
...